bolt-design-system/bolt
# Chore/move svg animations

## Jira

https://app.asana.com/0/1126340469288208/1137561697872889/f

## Summary

Move bolt-svg-animations into micro-journeys

## How to test

Build and check out the animations in Micro Journeys and at Experiments / Animations.
